gradle-plugin/src/main/java/io/neow3j/devpack/gradle/Neow3jCompileAction.java

```diff
@@ -0,0 +1,73 @@
+package io.neow3j.devpack.gradle;
+
+import static io.neow3j.devpack.gradle.Neow3jPluginUtils.getBuildDirURL;
+import static io.neow3j.devpack.gradle.Neow3jPluginUtils.getCompileOutputFileName;
+import static io.neow3j.devpack.gradle.Neow3jPluginUtils.getSourceSetsDirsURL;
+import static io.neow3j.devpack.gradle.Neow3jPluginUtils.writeToFile;
+import static java.nio.file.Files.createDirectories;
+import static java.util.Optional.ofNullable;
+
+import io.neow3j.compiler.Compiler;
+import io.neow3j.compiler.Compiler.CompilationResult;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.gradle.api.Action;
+
+public class Neow3jCompileAction implements Action<Neow3jCompileTask> {
+
+    @Override
+    public void execute(Neow3jCompileTask neow3jPluginCompile) {
+        String canonicalClassName = neow3jPluginCompile.getOptions().getClassName();
+
+        System.out.println("Smart Contract class: "
```

Comment:
Not sure how thorough i should do this review, but in the end this print line should be removed.